src/core/Pricing/PricingCards.tsx

```diff
@@ -240,20 +232,19 @@ const PricingCards = ({ data, delimiter }: PricingCardsProps) => {
                               ))}
                             </div>
                           ) : (
-                            <div key={item} className="flex gap-2 items-start">
+                            <div key={item} className="flex items-start">
```

Comment:
I don't think the padding around these icons is right when considering the design. Might be an alignment issue but there's more space around these in the design.